example/src/pages/index.js

```diff
@@ -25,25 +25,30 @@ class Index extends Component {
     // convert ms to s for gsap
     const seconds = length
 
-    return new TimelineMax()
+    return gsap.timeline()
       .set(this.transitionCover, { y: directionFrom })
-      .to(this.transitionCover, seconds / 2, {
+      .to(this.transitionCover, {
         y: '0%',
-        ease: Power1.easeInOut,
+        ease: "power1.easeInOut",
+        duration: seconds / 2,
       })
       .set(this.layoutContents, { opacity: 0 })
-      .to(this.transitionCover, seconds / 2, {
+      .to(this.transitionCover, {
         y: directionTo,
-        ease: Power1.easeIn,
+        ease: "power1.easeIn",
+        duration: seconds / 2,
       })
   }
 
   test(entry, node) {
-    return new TimelineMax().staggerFrom(
+    return gsap.to(
```

Comment:
This will reverse the original animation
```suggestion
    return gsap.from(
```

Reply:
Oops I guess I didn't test that one properly before committing. Updated now.